### ISSUE TITLE
Add anti-klepto to signing

### ISSUE THREAD

**Issue by prusnak** [trezor/trezor-firmware#2637]
Although Trezor code is fully open-source and builds are fully deterministic, this article provides good reason why to include the anti-klepto protocol to sec256k1 signing: https://shiftcrypto.ch/blog/how-almost-all-hardware-wallets-can-steal-your-seed/

Currently, the blocker is that this protocol currently does not support Schnorr signatures (=no Taproot), but this is to be addressed here: https://github.com/bitcoin-core/secp256k1/pull/1140

Once the above issue is resolved and the code is merged into the secp256k1 we might consider adding the anti-klepto feature.

**Comment by jadzeidan** [comment]
What about adding anti-klepto for non-taproot accounts in the meantime?
